sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice.json

```diff
@@ -0,0 +1,3093 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-03-13",
+    "title": "IotHub Gateway Service APIs"
```

Comment:
Is there a new swagger? doesn't the branch already contain one?  I see one here - https://github.com/Azure/azure-sdk-for-net/blob/feature/IoT-Hub/sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice.json